wd-java/src/test/java/com/example/InventoryTest.java

```diff
@@ -1,64 +1,78 @@
 package com.example;
 
+import static com.example.TestUtils.dotenv;
+
 import com.example.pageobjects.InventoryPage;
 import com.example.pageobjects.LoginPage;
+import com.saucelabs.visual.CheckOptions;
+import com.saucelabs.visual.CheckOptions.DiffingMethod;
 import com.saucelabs.visual.DataCenter;
 import com.saucelabs.visual.VisualApi;
 import com.saucelabs.visual.junit5.TestMetaInfoExtension;
+import com.saucelabs.visual.model.DiffingFlag;
+import java.net.MalformedURLException;
+import java.util.EnumSet;
 import org.junit.jupiter.api.AfterAll;
 import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.openqa.selenium.remote.RemoteWebDriver;
 
-import java.net.MalformedURLException;
-
-import static com.example.TestUtils.dotenv;
-
-
 @ExtendWith({TestMetaInfoExtension.class})
 public class InventoryTest {
 
-    private static final String username = dotenv.get("SAUCE_USERNAME");
-    private static final String accessKey = dotenv.get("SAUCE_ACCESS_KEY");
+  private static final String username = dotenv.get("SAUCE_USERNAME");
```

Comment:
double checking - since there is a lot of formatting changes, are you sure you didn't apply your own formatting rules to this file?

Reply:
I am probably the first person who ran `./mvnw com.spotify.fmt:fmt-maven-plugin:format` on the codebase. Unfortunately, it is not part of the CI, so past contributions didn't need to comply with it. I only committed changes to this file, but the whole examples could do with the same auto-formatting that we have in the SDKs repo. Out of scope for this MR though.